build-system/tasks/release/index.js

```diff
@@ -360,9 +360,20 @@ async function prependConfig_(outputDir) {
       type: channelConfig.type,
       ...require(`../../global-configs/${channelConfig.configBase}-config.json`),
     };
+    // Mapping of entry file names to a dictionary of AMP_CONFIG additions.
+    const targetsToConfig = MINIFIED_TARGETS.flatMap((minifiedTarget) => {
+      const targets = [];
+      if (!argv.esm) {
+        targets.push({file: `${minifiedTarget}.js`, config: {}});
+      }
+      if (argv.esm === undefined || argv.esm) {
```

Comment:
And I assume this covers `undefined` and `true`?

(Since the code doesn't make this obvious, can we add a comment here?)

Reply:
Correct. Added comments and a TODO!